setup_esptool.py

```diff
@@ -0,0 +1,3 @@
+import subprocess
+import sys
+subprocess.check_call([sys.executable, "-m", "pip", "install", "esptool", "setuptools"])
```

Comment:
Not a big fan of the fact that if this is run outside a virtualenv it will install python packages globally without that being very clear to the user.

Reply:
Fixed it using pipenv